lib/dependencies/RequireResolveDependencyParserPlugin.js

```diff
@@ -21,20 +21,24 @@ class RequireResolveDependencyParserPlugin {
 			if (expr.arguments.length !== 1) return;
 			const param = parser.evaluateExpression(expr.arguments[0]);
 			if (param.isConditional()) {
-				for (const option of param.options) {
+				param.options.forEach((option, idx) => {
```

Comment:
All these changes can be reverted as the expression is now tracked in the `BasicEvaluatedExpression`.